pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "codemagic-cli-tools"
-version = "0.40.4"
+version = "0.40.5"
```

Comment:
New feature should bump minor version, not only patch. The latter is for bugfixes and marginal improvements.